sway-lib-std/src/math.sw

```diff
@@ -2,7 +2,12 @@
 library;
 
 use ::assert::*;
-use ::flags::{disable_panic_on_overflow, F_UNSAFEMATH_DISABLE_MASK, set_flags};
+use ::flags::{
+    disable_panic_on_overflow,
+    F_UNSAFEMATH_DISABLE_MASK,
+    F_WRAPPING_DISABLE_MASK,
+    set_flags,
```

Comment:
```suggestion
```
Unused import

Reply:
That was not added in this pr, it is used in logarithm for u256